src/components/views/settings/SetIdServer.js

```diff
@@ -260,17 +260,21 @@ export default class SetIdServer extends React.Component {
 
         const boundThreepids = threepids.filter(tp => tp.bound);
         let message;
+        let danger = false;
         if (boundThreepids.length) {
             message = _t(
-                "You are currently sharing email addresses or phone numbers on the identity " +
-                "server <idserver />. You will need to reconnect to <idserver2 /> to stop " +
-                "sharing them.", {},
+                "You are still <b>sharing your personal data</b> on the identity " +
+                "server <idserver />. <br /> <br /> " +
```

Comment:
not sure if we should have the br's in the translation: can we just use two paragraphs instead?

Reply:
It seems we can't currently use `<p>` here because the translation utils don't support tags within tags (for the `<b>` inside the `<p>`).

We could make a custom dialog component here, but it feels like overkill to add it just for this reason.

Reply:
Yes, I changed to `<p>`s outside the strings... Not sure why it didn't occur to me to try that the first time! 😅